src/Controller/SentencesController.php

```diff
@@ -555,8 +555,10 @@ public function search()
     public function advanced_search() {
         $search = new SentencesSearchForm();
 
-        $search->setData([]);
-        $this->set($search->getData());
+        $search->setData($this->request->getQueryParams());
+        $usesTemplate = $search->getData() != SentencesSearchForm::DEFAULT_CRITERIA;
```

Comment:
@jpear1 Thanks answering my comments! I saw you changed the default criteria into a const. I’d like to suggest a more object-oriented approach. You can keep this attribute private and add a new method to `SentencesSearchForm` to check if the form is using default values or not. If you choose a good name for that method, the above line could be easier to understand. For example:
```php
$usesTemplate = !$search->isUsingDefaultCriteria();
```